docs/preview/yamlgettingstarted-triggers.md

```diff
@@ -0,0 +1,96 @@
+# YAML getting started - YAML triggers (not yet available, for discussion only)
+
+## Simple CI trigger syntax
+
+On push, the YAML file from the branch that was updated, is used to determine whether
+to queue a CI build.
+
+For example, if master is updated, the trigger defined in master will be used. If a different
+branch is updated, the YAML file from that branch will be used to determine whether to queue a
+build.
+
+```yaml
+trigger:
+- master
+- releases/*
+```
+
+## CI is opt-out
+
+When not specified, the default trigger for build would be all branches.
+
+For build this behavior makes sense. Especially when we start supporting on-push defintion creation from any branch. And on-push resource authorization from any branch.
+
+For release, CI trigger would likely be opt-in? However with YAML, release will be tightly coupled with a repository. So opt-out CI behavior might also make sense for release? Or opt-out, but the default for RM only includes master?
```

Comment:
I'm not sure about whether it should be opt-in/out. One thing to note is that release branches are often merged into other branches (especially master) as servicing fixes are created.   This means yaml files are merged around a lot. Mistakenly enabling CI for a branch because of a merge is probably not a big deal, but mistakenly enabling release might be more significant.

Reply:
In this case we are talking about the VSTS Release project not specifically release branches.  

We have been discussing if the absence of any specified trigger pattern should mean trigger on all branches by default.